src/algebra/ring/aut.lean

```diff
@@ -29,7 +30,7 @@ ring_aut
 @[reducible] def ring_aut (R : Type*) [has_mul R] [has_add R] := ring_equiv R R
 
 namespace ring_aut
-variables (R : Type*) [has_mul R] [has_add R]
+variables (R G : Type*) [has_mul R] [has_add R] [group G]
```

Comment:
To elaborate on my comment below:

```suggestion
section mul_add
variables (R G : Type*) [has_mul R] [has_add R] [group G]
```
and then a new `section semiring` with all the new stuff in. Don't forget to `end mul_add`